config/application.rb

```diff
@@ -14,13 +14,15 @@
 
 
 # Engine loading mechanism
+# :nocov:
```

Comment:
For the reviewer:

`application.rb` will fail with code coverage with the core suite unless we ignore these lines. We can only check coverage with the engines suite.

Reply:
Oh yeah, that's probably what `fail_on_error = false` was for.